specification/vmwarevirtustream/resource-manager/Microsoft.VMwareVirtustream/preview/2016-09-01-preview/vmwarevirtustream.json

```diff
@@ -0,0 +1,1304 @@
+            "schema": {
+              "$ref": "#/definitions/PrivateCloudResponseList"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "PrivateClouds_List": {
+            "$ref": "./examples/PrivateClouds_List.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.VMwareVirtustream/privateClouds/{privateCloudName}": {
+      "get": {
+        "operationId": "PrivateClouds_GetById",
+        "tags": [
+          "PrivateClouds"
+        ],
+        "summary": "get a private cloud",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "name": "privateCloudName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the private cloud."
+          },
+          {
+            "$ref": "#/parameters/apiVersion"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "successful operation",
+            "schema": {
+              "$ref": "#/definitions/PrivateCloudResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "PrivateClouds_GetById": {
+            "$ref": "./examples/PrivateClouds_GetById.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.VMwareVirtustream/privateClouds": {
+      "get": {
+        "operationId": "PrivateClouds_ListInResourceGroup",
+        "tags": [
+          "PrivateClouds"
+        ],
+        "summary": "list private clouds in a resource group",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/apiVersion"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "successful operation",
+            "schema": {
+              "$ref": "#/definitions/PrivateCloudResponseList"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "PrivateClouds_ListInResourceGroup": {
+            "$ref": "./examples/PrivateClouds_ListInResourceGroup.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.VMwareVirtustream/privateClouds/{privateCloudName}": {
+      "get": {
+        "operationId": "PrivateClouds_GetByIdInResourceGroup",
+        "tags": [
+          "PrivateClouds"
+        ],
+        "summary": "get a private cloud in a resource group",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupName"
+          },
+          {
+            "name": "privateCloudName",
```

Comment:
Might be worth declaring this in a common place (similar to the 'subscriptionId' and 'resourceGroupName' parameters) to avoid repetition.

Reply:
I added it as a parameter and used references everywhere.